backend/de.metas.postfinance.base/src/main/sql/postgresql/system/90-postfinance/5741260_sys_config_postFinanceInvoiceBatchSize.sql

```diff
@@ -0,0 +1,14 @@
+-- Run mode: WEBUI
+
+-- SysConfig Name: de.metas.postfinance.document.export.PostFinanceYbInvoiceService.PostFinanceUploadInvoiceBatchSize
+-- SysConfig Value: 40
+-- 2024-12-06T11:40:20.001Z
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (1000000,1000000,541748,'O',TO_TIMESTAMP('2024-12-06 12:40:19.983','YYYY-MM-DD HH24:MI:SS.US'),100,'U','Y','de.metas.postfinance.document.export.PostFinanceYbInvoiceService.PostFinanceUploadInvoiceBatchSize',TO_TIMESTAMP('2024-12-06 12:40:19.983','YYYY-MM-DD HH24:MI:SS.US'),100,'20')
```

Comment:
please ad add description so I know that a batch basically consists of invoices with their PDFs. Also add the known size restrictions